### DISCUSSION TITLE
No reply from GitHub on ticket (#2697349)

### DISCUSSION THREAD

**Discussion by 00nodetest** [community/community#127274] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

My account was compromised and suspended I raised request to restore it and sent multiple emails still haven't heard from GitHub, it's been months.

**Comment by ebndev** [comment]
Hi @00nodetest,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
